ds-prep-python-lists/main.py

```diff
@@ -0,0 +1,37 @@
+ages = [29, 19, 17, 26, 32, 16, 21]
+print(ages)
+
+print(ages[4])
+
+print(ages[-2])
+
+print(ages[2:6])
+
+print(ages[:3])
+
+if 17 in ages:
+  print(True)
+else:
+  print(False)
+
+if 42 in ages:
+  print(True)
+else:
+  print(False)
+
+ages[2] = 18
+print(ages)
+
+temp = ages[4]
+ages[4] = ages[3]
+ages[3] = temp
+print(ages)
+
+ages.append(45)
+ages.insert(0, 32)
+ages.insert(6, 37)
```

Comment:
This is just before the seventh value, not the sixth:

---

![image](https://user-images.githubusercontent.com/16143771/146114808-ba3a348a-9a04-4af4-b7e9-2fc752814cd7.png)